apps/framework-cli/src/cli/watcher.rs

```diff
@@ -325,17 +326,11 @@ async fn watch(
     while let Some(res) = rx.recv().await {
         match res {
             Ok(debounced_events) => {
-                let events = debounced_events
-                    .into_iter()
-                    .filter(|event| !event.kind.is_access() && !event.kind.is_other())
```

Comment:
So do we filter out `access` events in the new implementation?

Reply:
It bundles events with at most 1 event a second - which included access events, etc...
That reduces the amounts of refreshes drastically.

Reply:
I got this simply by opening a flow file with `vi`

```
⢺ Processing 2 Flow(s) changes from file watcher
⢺ Processing 1 Flow(s) changes from file watcher
⢺ Processing 1 Flow(s) changes from file watcher
```

Might not be a big deal though.